contribs/gmf/src/directives/layertree.js

```diff
@@ -320,15 +321,34 @@ gmf.LayertreeController.prototype.getLayerCaseMixedGroup_ = function(node) {
  * @private
  */
 gmf.LayertreeController.prototype.getLayerCaseNotMixedGroup_ = function(node) {
-  var names = this.retrieveNodeNames_(node, true);
+  var childs = [];
+  this.getFlatNodes_(node, childs);
+  var layersNames = childs.map(function(node) {
+    return node['layers'];
+  }).join(',');
   var url = node.url || this.gmfWmsUrl_;
   var serverType = node.children[0]['serverType'];
-  var layer = this.layerHelper_.createBasicWMSLayer(url, '', serverType);
-  this.updateWMSLayerState_(layer, names);
+  var timeParam;
+  var timeNode;
 
+  if ((node.time && node.time['minValue']) || (timeNode = childs.filter(hasTime)).length) {
```

Comment:
Oh. I see. Can you please move the variable value affection outside the if condition, for readability?